src/debug/di/process.cpp

```diff
@@ -9742,6 +9742,11 @@ void CordbProcess::MarshalManagedEvent(DebuggerIPCEvent * pManagedEvent)
 //    The event still needs to be Marshaled before being used. (see code:CordbProcess::MarshalManagedEvent)
 //
 //---------------------------------------------------------------------------------------
+#ifdef _TARGET_ARM_
+// This is a temporary workaround for an ARM specific C++ compiler bug.
```

Comment:
What kind of compiler bug? And which compiler? The comment should state that. And I would add TODO: in front of the "It should be removed once the bug is fixed" so that it is easier to locate. Otherwise I am afraid it would stay there forever.